docs/self-managed/identity/user-guide/managing-user-access.md

```diff
@@ -0,0 +1,45 @@
+---
+id: managing-user-access
+title: "Managing user access to Identity"
+sidebar_label: "Managing user access"
+---
+
+In this guide, you will learn how to grant users different levels of access to the Identity application.
+
+## Permissions supported by Identity
+
+Identity implements the following permissions:
+
+- `read`: Users can access all pages in Identity. They _cannot_ create, modify, or delete any data.
+- `read:users`: Users can access only the **Users** page and related subpages.
+- `write`: Users have access to all pages. They can create, modify, and delete data.
+
+:::note
+If you are connecting Identity to your existing Keycloak installation, the `read:users` permission needs to be created
+manually in Keycloak.
+:::
+
+You can [assign the above permissions to users as part of a role](assigning-a-role-to-a-user.md).
+This gives the user access to the Identity application.
+
+## Assign Identity permissions to a user
+
+Without an Identity permission, users can access Identity only to log in to other applications.
```

Comment:
I wonder if this paragraph can be simplified to
"Users are unable to access the Identity UI without at least one of the permissions listed above. To grant a user access to the Identity UI ..."

Reply:
I initially wanted to write something similar. The reason behind mentioning the login (even though its functionality is part of Keycloak) was to make sure users understand that users can still log in using Identity. This is under the assumption that customers use our Keycloak with our theme in which case they will likely not know that it is part of Keycloak and not Identity. Wdyt?

I like that you referenced the permissions mentioned above, this makes it less ambiguous 👍

Reply:
Ah fair point, I think thats a fair suggestion to make actually as Users may not be aware that the login flows still happen via Identity its just the UI that is protected by permissions. An alternative suggestion could be 

"Users are always able to use Identity to login to the components however they are unable to access the Identity UI without at least one of the permissions listed above. To grant a user access to the Identity UI ..."

Reply:
I like your suggestion 👍